redaxo/src/core/lib/setup/setup.php

```diff
@@ -156,32 +156,32 @@ public static function checkDb($config, $createDb)
     /**
      * Basic php security checks. Returns a human readable strings on error.
      *
-     * @return string
+     * @return string[]
```

Comment:
@bloep muss noch in console übernommen werden

Reply:
Ping

Reply:
Wenn ich das richtig gesehen habe, hast du das doch schon hier im PR mit gemacht ?

Reply:
War mir nicht sicher was dein Kommentar genau meint :-). 

Wenns nix zu tun gibt - umso besser